src/components/cart/OrderSummary/order-summary.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../../styles/scaffold";
+
+.order-summary[data-order-summary] {
+  padding: 1rem;
+
+  [data-order-summary-subtotal] {
+    display: flex;
+    justify-content: space-between;
+    line-height: 1.5;
+  }
+
+  [data-order-summary-discount] {
+    display: flex;
+    justify-content: space-between;
+    color: green;
```

Comment:
There should be some CSS custom property we can use here (e.g. `color: var(--color-text-discount)`). 

Do we already have that? cc @hellofanny and @renatamottam

Reply:
We don't! 

Dan, can you please add this new variable?
- on `globals.scss` file (on the // Contextual Colors section), please add `--color-success-dark: #368369`
- on `colors.scss` file  (on the   // Text section), please add `--color-text-success: var(--color-success-dark)`

And then, replace `green` for `var(--color-text-success)`

@renatamottam agree?

Reply:
Thanks, @hellofanny. It's done like you said :D